### PR TITLE
Add Oracle SQL - ANALYZE

### PR DESCRIPTION
**Pull request by Liangda-w**
Relates #10513 

Changes proposed in this pull request:
-  Add Oracle SQL - [ANALYZE](https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/ANALYZE.html#GUID-535CE98E-2359-4147-839F-DCB3772C1B0E)
-  Add corresponding test cases

@wgy8283335

### CODE REVIEW

File: shardingsphere-sql-parser/shardingsphere-sql-parser-engine/src/main/java/org/apache/shardingsphere/sql/parser/core/visitor/SQLVisitorRule.java [hunk 1]
```diff
@@ -108,6 +108,8 @@
     
     DROP_VIEW("DropView", SQLStatementType.DDL),
 
+    ANALYZE("Analyze", SQLStatementType.DDL),
```
Comment (wgy8283335) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-engine/src/main/java/org/apache/shardingsphere/sql/parser/core/visitor/SQLVisitorRule.java:111]: Why set `Analyze` as DDL?

File: shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/asserts/statement/ddl/DDLStatementAssert.java [hunk 2]
```diff
@@ -87,6 +90,8 @@ public static void assertIs(final SQLCaseAssertContext assertContext, final DDLS
             AlterSessionStatementAssert.assertIs(assertContext, (OracleAlterSessionStatement) actual, (AlterSessionStatementTestCase) expected);
         } else if (actual instanceof AlterSystemStatement) {
             AlterSystemStatementAssert.assertIs(assertContext, (AlterSystemStatement) actual, (AlterSystemStatementTestCase) expected);
+        } else if (actual instanceof OracleAnalyzeStatement) {
```
Comment (wgy8283335) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/asserts/statement/ddl/DDLStatementAssert.java:93]: Are you sure that `analyze` should be put in the DDL？

**Review by wgy8283335 — CHANGES_REQUESTED** [review]
The `Analyze` should be classified as DDL or DAL?

**Comment by Liangda-w** [comment]
> The `Analyze` should be classified as DDL or DAL?

Hi @wgy8283335 I thought it will be DDL because of the [Oracle Doc](https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/Types-of-SQL-Statements.html#GUID-FD9A8CB4-6B9A-44E5-B114-EFB8DA76FC88):

image

But you're right, in [MySQL](https://dev.mysql.com/doc/refman/8.0/en/analyze-table.html) `ANALYZE` is classified as DAL. So do you think change to DAL is better?

**Comment by wgy8283335** [comment]
> > The `Analyze` should be classified as DDL or DAL?
> 
> Hi @wgy8283335 I thought it will be DDL because of the [Oracle Doc](https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/Types-of-SQL-Statements.html#GUID-FD9A8CB4-6B9A-44E5-B114-EFB8DA76FC88):
> 
> image
> 
> But you're right, in [MySQL](https://dev.mysql.com/doc/refman/8.0/en/analyze-table.html) `ANALYZE` is classified as DAL. So do you think change to DAL is better?

I think DDL is better.

**Review by wgy8283335 — APPROVED** [review]

**Pull request merged**